LocalMultiplayerAgent/Config/MultiplayerSettingsValidator.cs

```diff
@@ -35,9 +35,9 @@ public bool IsValid()
                 throw new Exception("OutputFolder or TitleId not found. Call SetDefaultsIfNotSpecified() before this method");
             }
 
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && _settings.RunContainer)
```

Comment:
this will give "true" if the OS is MacOS. Do we want to support that at this time? This "may just work" ([https://github.com/PlayFab/MpsAgent/pull/30](https://github.com/PlayFab/MpsAgent/pull/30)) but not sure till we test it out

Reply:
Thanks Dimitris, I will test it out and see if we need to consider MacOS case!

Reply:
I will add **if RuntimeInformation.IsOSPlatform(OSPlatform.OSX) them throw some exception with message** because we still don't know if we are supping OS or not. any thought? :)

Reply:
that's great - I would even add "we would be happy to accept PRs to make it work" :)